docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,71 @@
+Canonical URLs
+==============
+
+A `Canonical URL`_
+allows you to specify the preferred version of a web page to prevent duplicated content.
+They are mainly used by search engines to link users to the correct
+version and domain of your documentation.
+
+If canonical URL's aren't used,
+it's easy for outdated documentation to be the top search result for various pages in your documentation.
+This is not a perfect solution for this problem,
+but generally people finding outdated documentation is a big problem,
+and this is one of the suggested ways to solve it from search engines.
+
+.. _Canonical URL: https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls
+
+How Read the Docs generates canonical URLs
+------------------------------------------
+
+The canonical URL takes into account:
+
+* The default version of your project (usually "latest" or "stable").
+* The canonical :doc:`custom domain </custom-domains>` if you have one,
+  otherwise the default :ref:`subdomain <hosting:subdomain support>` will be used.
+
+For example, if you have a project named ``example-docs``
+with a custom domain ``https://docs.example.com``.
+A search engine like Google will index both domains, ``https://example-docs.readthedocs.io`` and
+``https://docs.example.com``.
+
+You'll want to use ``https://docs.example.com`` as your canonical domain,
+this means that when Google indexes a page like ``https://example-docs.readthedocs.io/en/latest/``,
```

Comment:
```suggestion
This means that when Google indexes a page like ``https://example-docs.readthedocs.io/en/latest/``,
```

Reply:
@agjohnson You can do multi-line suggestions in cases like this to make review easier.